src/js/common/components/PrivacyBody.jsx

```diff
@@ -84,7 +84,7 @@ export default class PrivacyBody extends Component {
             programs, and to send you timely election reminders.
             You may opt-out of receiving messages from WeVote by sending any email to
             {' '}
-            <a href="mailto:info@WeVote.US" target="_blank" rel="noopener noreferrer">info@WeVote.US</a>
+            <a href="mailto:info@WeVote.US" target="_blank" class="class u-link-color" rel="noopener noreferrer">info@WeVote.US</a>
```

Comment:
Hi @Almee98 thank you for pushing this issue forward! I notice this is where the extra "class " is that I mentioned in our meeting yesterday. (It can be removed)